src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -2880,27 +2878,19 @@ public Void emitObjectType(
      * If a symbol refers to a "platform" type (e.g. Promise) emit it specially here and return
      * true.
```

Comment:
Change function description. Specifically call out that it may return null if not a platform symbol.

Reply:
Done.